src/components/screens/monitor/delegates/withForgingStatus.js

```diff
@@ -199,6 +200,8 @@ const withForgingStatus = delegatesKey => (ChildComponent) => {
             ...this.props[delegatesKey],
             data: this.getDelegatesData(),
           },
+          lastBlock: latestBlocks[0],
```

Comment:
🐛 Unfortunately, this works only when the page is loaded and `latestBlocks[0]` is from Lisk Service API, but once there's a new block received from  the Lisk Core socket, this one doesn't contain `generatorUsername`

Reply:
Just fixed with https://github.com/LiskHQ/lisk-desktop/pull/2641/commits/1176f1e021ffe4825834713046d8e2a7edc06f1f